docs/reference/method/MongoDBGridFSBucket-drop.txt

```diff
@@ -26,4 +26,38 @@ Errors/Exceptions
 
 .. include:: /includes/extracts/error-driver-runtimeexception.rst
 
-.. todo: add examples
+Examples
+--------
+
+.. code-block:: php
+
+   <?php
+
+   $database = (new MongoDB\Client)->test;
+
+   $bucket = $database->selectGridFSBucket();
+
+   $stream = fopen('php://temp', 'w+b');
+   fwrite($stream, "foobar");
+   rewind($stream);
+
+   $bucket->uploadFromStream('filename', $stream);
+
+   $bucket->drop();
+
+   $collections = $database->listCollections();
+
+   $foundCollection = null;
+
+   foreach ($collections as $collection) {
+       if ($collection->getName() === $collectionName) {
+           $foundCollection = $collection;
+           break;
+       }
+   }
+
+   var_dump($foundCollection);
```

Comment:
Let's get rid of everything after `$bucket->drop()` and the expected output, as we did with `delete()`.